pkgs/top-level/dotnet-packages.nix

```diff
@@ -326,14 +326,14 @@ let self = dotnetPackages // overrides; dotnetPackages = with self; {
 
   Boogie = buildDotnetPackage rec {
     baseName = "Boogie";
-    version = "2019-06-20";
-    name = "${baseName}-unstable-${version}";
+    version = "2.4.1";
+    name = "${baseName}-${version}";
```

Comment:
`pname`, maybe?

Reply:
well, baseName amounts to pname in dotnetPackages, but `name = "${baseName}-${version}";` is completely redundant with buildSotnetPackage indeed.